indigo/distribution.yaml

```diff
@@ -4472,6 +4472,32 @@ repositories:
       url: https://github.com/ros-industrial/industrial_metapackages.git
       version: indigo
     status: developed
+  industrial_moveit:
+    doc:
+      type: git
+      url: https://github.com/ros-industrial/industrial_moveit.git
+      version: indigo
+    release:
+      packages:
+      - constrained_ik
+      - industrial_collision_detection
+      - industrial_moveit
+      - industrial_moveit_benchmarking
+      - stomp_core
+      - stomp_moveit
+      - stomp_plugins
+      - stomp_test_kr210_moveit_config
+      - stomp_test_support
+      tags:
+        release: release/indigo/{package}/{version}
+      url: https://github.com/ros-industrial-release/industrial_moveit-release.git
+      version: 0.1.0-0
+    source:
+      test_pull_requests: true
```

Comment:
Have you made sure that the Jenkins GitHub user has the necessary access to this org unit (see http://wiki.ros.org/buildfarm/Pull%20request%20testing)?